src/js/control-bar/volume-menu-button.js

```diff
@@ -58,13 +58,42 @@ class VolumeMenuButton extends MenuButton {
     updateVisibility.call(this);
     this.on(player, 'loadstart', updateVisibility);
 
-    this.on(this.volumeBar, ['slideractive', 'focus'], function(){
-      this.addClass('vjs-slider-active');
-    });
+    this.toggleMutedLabel();
 
-    this.on(this.volumeBar, ['sliderinactive', 'blur'], function(){
-      this.removeClass('vjs-slider-active');
+    this.on('focus', function() {
+      this.el_.setAttribute('aria-expanded', 'true');
     });
+
+    let playerEl = player.el();
+    let handleMouseOver = (event) => {
+      if (event.target !== this.el_ && event.target !== this.volumeBar.el_) {
+        this.el_.setAttribute('aria-expanded', 'false');
+      }
+    };
+    playerEl.onfocusin = handleMouseOver;
+    playerEl.addEventListener('focus',handleMouseOver,true);
+    playerEl.onfocusout = handleMouseOut;
+    playerEl.addEventListener('blur',handleMouseOut,true);
```

Comment:
This block needs to be cleaned up. I think all the `handleMouseOut` can be removed. Also, it needs to be removed on `dispose`.